app/src/shared/utils/passiveEventOptions.js

```diff
@@ -0,0 +1,21 @@
+export default (() => {
+    let options = false
+
+    if (typeof window !== 'undefined') {
+        try {
+            window.addEventListener('test', null, Object.defineProperty({}, 'passive', {
```

Comment:
What does this do? Just curious 🤔

Reply:
Safely detects `passive` event handlers support. See MDN's ["Improving scrolling performance with passive listeners"](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Improving_scrolling_performance_with_passive_listeners) for details. I'll include a note about it in the code.